docs/adr/0014-signing-release-packages.md

```diff
@@ -0,0 +1,82 @@
+# 14) Signing release packages {#adr_0014}
+
+Date: 2021-05-03
+
+## Status
+
+proposed
+
+## Context
+
+During the course of release strategy hike a brief [investigation](https://phabricator.wikimedia.org/T272418) into signing of release packages was done.
+
+Signing the artifacts our release pipeline produces is a step that can be taken to ensure their source and integrity. Our pipeline produces two different kinds of artifacts that both have the possibility to have a signature.
+
+- Tarball packages
+- Docker images
+
+### Signing tarball packages with GPG
+
+Tarball files can be signed with [GPG](https://gnupg.org/gph/en/manual/x135.html) in a similar way to which MediaWiki are signing theirs. In the case of MediaWiki a private and public key-pair is generated for release engineering members and published on https://www.mediawiki.org/keys/keys.html. 
+
+By importing the list of public keys the end-user can then verify the release tarball and it's signature by issuing the following command.
+
+```
```

Comment:
Maybe mention that `gpg --fetch-keys "https://www.mediawiki.org/keys/keys.txt"` needs to run first?

Reply:
done